src/sentry/tasks/relocation.py

```diff
@@ -63,6 +62,7 @@
 from sentry.silo.base import SiloMode
 from sentry.tasks.base import instrumented_task
 from sentry.types.region import get_local_region
+from sentry.users.models.lostpasswordhash import LostPasswordHash as LostPasswordHash
```

Comment:
```suggestion
from sentry.users.models.lostpasswordhash import LostPasswordHash
```
We don't need to alias this to the same name.